### PR TITLE
Contrib PoC script for issue #503 and README updates

### PR DESCRIPTION
**Pull request by jimklimov**
The script is not production quality and does not even cover all the cases I've seen, but on one hand it allows to rectify some of the problems with simple single-dataset ZBEs already (running it now locally to fix my destination of zones' backups), and on another can serve as a basis for proper implementation inside znapzend itself.

The updated README.md and its markup can be seen rendered from markdown e.g. in my branch source for this PR at https://github.com/jimklimov/znapzend/tree/contrib-issue-503

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,41 +40,54 @@ On RedHat you get the necessaries with:
 
 On Ubuntu / Debian with:
 
-    apt-get install perl unzip
+    apt-get install perl unzip autoconf carton
```
Comment (oetiker) [inline README.md:43]: hmmm autoconf is only required when building the git checkout ... carton should never be required

Comment (jimklimov) [reply README.md:43]: Hmm... I always started with bootstrap (unlike the README instructions) so indeed autoconf may be an overshot. Apparently this (both) is also what @Dacesilian used on Gitter.

No opinion about carton, except that maybe using an OS-provided one is somehow better for support/compat/etc?

Comment (jimklimov) [reply README.md:43]: Curious, the release zip file does not include `configure` script, but the tarball does.
It is both tracked in git SCM and excluded in `.gitignore`, if that helps.

Comment (jimklimov) [reply README.md]: Amended the README to convey that bootstrap exists and then might need autoconf and family to be used.

**Pull request merged**